com.unity.render-pipelines.high-definition/Documentation~/3ds-physical-shader.md

```diff
@@ -1,41 +1,131 @@
 
-Note that the HDRP implementation of this shader uses a Shader Graph.
+**Note**: There are slight differences between what you see in Autodesk® 3DsMax and what you see in HDRP. HDRP doesn't support some material features.
+
+**Note**: The HDRP implementation of the 3DS Physical Material shader uses a Shader Graph.
 
 ## Creating a 3DS Physical Material
 
-When Unity imports an FBX with a compatible 3DS Physical Material shader, it automatically creates a 3DS Physical Material. If you want to manually create an 3DS Physical Material:
+When Unity imports an FBX with a compatible 3DS Physical Material shader, it automatically creates a 3DS Physical Material. If you want to manually create a 3DS Physical Material:
 
-1. Create a new material (menu: **Assets > Create > Material**).
-2. In the Inspector for the Material, click the Shader drop-down then click **HDRP > 3DSMaxPhysicalMaterial > PhysicalMaterial3DSMax**.
+1. Create a new material (menu: **Assets** > **Create** > **Material**).
+2. In the Inspector for the Material, click the Shader drop-down then click **HDRP** > **3DSMaxPhysicalMaterial** > **PhysicalMaterial3DSMax**.
 
 ## Properties
 
 ### Surface Options
+
 **Surface Options** control the overall look of your Material's surface and how Unity renders the Material on screen.
+
 <table>
-<tr>
-<th>Property</th>
-<th>Description</th>
-</tr>
-[!include[](snippets/shader-properties/surface-options/surface-type.md)]
-[!include[](snippets/shader-properties/surface-options/rendering-pass.md)]
-[!include[](snippets/shader-properties/surface-options/blending-mode.md)]
-[!include[](snippets/shader-properties/surface-options/preserve-specular-lighting.md)]
-[!include[](snippets/shader-properties/surface-options/sorting-priority.md)]
-[!include[](snippets/shader-properties/surface-options/receive-fog.md)]
-[!include[](snippets/shader-properties/surface-options/transparent-depth-prepass.md)]
-[!include[](snippets/shader-properties/surface-options/transparent-writes-motion-vectors.md)]
-[!include[](snippets/shader-properties/surface-options/depth-write.md)]
-[!include[](snippets/shader-properties/surface-options/depth-test.md)]
-[!include[](snippets/shader-properties/surface-options/cull-mode.md)]
-[!include[](snippets/shader-properties/surface-options/double-sided.md)]
-[!include[](snippets/shader-properties/surface-options/normal-mode.md)]
-[!include[](snippets/shader-properties/surface-options/receive-decals.md)]
-[!include[](snippets/shader-properties/surface-options/receive-ssr.md)]
-[!include[](snippets/shader-properties/surface-options/receive-ssr-transparent.md)]
+<thead>
+  <tr>
+    <th>Property</th>
+    <th></th>
+    <th></th>
+    <th>Description</th>
+  </tr>
```

Comment:
why are you expanding all the snippet? Those snippet are reuse at various place, and we don't want to have to patch every file when doind a correction